modules/@apostrophecms/schema/index.js

```diff
@@ -1828,6 +1828,7 @@ module.exports = {
               await self.apos.util.recursionGuard(req, `${_relationship.type}:${_relationship.withType}`, () => {
                 // Allow options to the getter to be specified in the schema,
                 // notably editable: true
+                // TODO editable going away in favor of disabled for consistency
```

Comment:
@boutell I found this note while normalizing relationship field's disabled states, I don't think we use this `editable` property anymore, is there anything to be done here?

Reply:
It's dead, you can drop the note.